.eslintrc.json

```diff
@@ -42,6 +44,10 @@
         "eol-last": 2,
         "eqeqeq": 2,
         "header/header": [2, "block", [ "", " * Copyright (c) Microsoft Corporation. All rights reserved.", " * Licensed under the MIT License.", " " ], 2],
+        "jasmine/no-disabled-tests": 1,
+        "jasmine/no-focused-tests": 2,
+        "jest/no-disabled-tests": 1,
+        "jest/no-focused-tests": 2,
```

Comment:
Do we need to add mocha as well?

Reply:
^^

Reply:
@jasonnutter Can we add mocha to this? I will work on migrating common and browser to jest at some point this quarter.

Reply:
Don't need both, as they both use the same grammar for skipped and disabled tests.